python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -81,7 +81,32 @@ def _add_fake_sources(exposure, objects, calibFluxRadius=12.0, logger=None):
         mat = wcs.linearizePixelToSky(spt, geom.arcseconds).getMatrix()
         gsWCS = galsim.JacobianWCS(mat[0, 0], mat[0, 1], mat[1, 0], mat[1, 1])
 
-        psfArr = psf.computeKernelImage(pt).array
+        try:
+            psfArr = psf.computeKernelImage(pt).array
+        except InvalidParameterError:
+            logger.infof("Cannot compute Psf for object at {}; skipping", pt)
```

Comment:
The log here is a bit of a misnomer as it the object isn't being skipped just yet assuming that the new point creation doesn't return the same position.